bot.py

```diff
@@ -56,19 +57,215 @@
     embed.clear_fields()
 
 
+async def displayEmbedCountries(message, playerNumber, countries):
+    global hoi_dict
+    embed = discord.Embed(
+        title='Player ' + str(playerNumber + 1),
+        colour=random.randint(0, 0xffffff)
+    )
+    allImages = []
+    allCountries = []
+    for cou in hoi_dict:
+        for idcountry in countries:
```

Comment:
This is a great example of where you could pick better variable names:
`for country in countries`...
country is very clearly a single entity out of countries.